src/test/resources/pages/studentHomeHTML.html

```diff
@@ -31,6 +31,29 @@ <h1>
   </div>
   <br>
   <br>
+  <div class="panel panel-primary">
```

Comment:
Hm that's very strange. Perhaps it was inserted from another merged PR when I updated the branch earlier, because I definitely did not insert this part through my changes